LondonBritishLibrary/orient/BLorient13156.xml

```diff
@@ -103,14 +103,27 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <measure unit="leaf">161</measure>
                                     <measure unit="page" type="blank">4</measure><locus target="#46r #112r #127r #144r"/>
                                     <dimensions type="outer" unit="mm">
+                                        <height>113</height>
+                                        <width>98</width>
+                                        <depth>63</depth>
+                                    </dimensions>
+                                    <dimensions xml:id="leafdim" type="outer" unit="mm">
                                         <height>115</height>
                                         <width>92</width>
-                                        <depth/>
                                     </dimensions>
                                 </extent>
                             </supportDesc>
                             <layoutDesc>
-                                <layout columns="1" writtenLines="12 15"/>
+                                <layout columns="1" writtenLines="12 15">   
+                                </layout>
+                                <layout>
+                                    <locus target="#4v #46v #62v #95v #112v #127v #131v #144v"/>
+                                    <note corresp="#d1 #d2 #d3 #d4 #d5 #d6 #d7 #d8">The illustrated folios have a different ruling pattern from the written
```

Comment:
In order not to plagiarize, I think we need to mention the source when quoting (almost) verbatim. What is the Bm practice on this?

Reply:
I think we have a legion of verbatim quotes from catalogues, Wikipedia or other websites in our records, with the source cited at the end in <additional> or in <bibliography>, or (in case of wikipedia) not cited at all.

Reply:
If I understand you correctly, I don't think this is a good practice. The reader must be able to tie specific information to specific sources, no?

 @eu-genia @DenisNosnitsin1970

Reply:
When a source is quoted then the bibliography should be provided at the quotation. Catalogues which are the basis for the description are different, it is assumed that they are the main source of information, so it is even more important to specify the bits that come from elsewhere.